src/neuroconv/datainterfaces/ecephys/requirements.txt

```diff
@@ -1 +1,2 @@
-spikeinterface>=0.100.0
+spikeinterface>=0.100.1,<0.100.6
+neo>=0.13.1
```

Comment:
But then for release purposes won't we _need_ `spikeinterface>=0.100.6`?

Reply:
I don't think so.
But, once they merge this https://github.com/SpikeInterface/spikeinterface/pull/2833 or they release probe interface then the ceiling on the version can be removed.